cfpb/consumerfinance.gov
# Adding back Prepaid agreements tests

adding back the prepaid agreements tests and updating the test database for them.


## Additions

- one more test to our functional tests on GHA



## Changes

- the test database now has the data for the agreements tests


## How to test this PR

1. look into the logs of the functional test that should have run and check for a test called prepaid-agreements-search.js



## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
